lib/instance-validator.js

```diff
@@ -108,7 +108,7 @@ InstanceValidator.prototype._builtinValidators = function() {
   var validators = [];
   Utils._.forIn(this.modelInstance.rawAttributes, function(rawAttribute, field) {
     if (self.options.skip.indexOf(field) >= 0) {
-      return;
+      return false;
```

Comment:
https://github.com/lodash/lodash/blob/master/lodash.js#L3352 it tests for explicit false, `return undefined` would not break loop.